tests/structures/test_exception.py

```diff
@@ -45,3 +45,21 @@ def test_raise_inside_loop(self):
                 except:
                     print(x)
         """)
+
+    def test_repr(self):
```

Comment:
With the unexpected behaviour of `StopIteration.__str__`, maybe we could use a copy of this test with`repr` changed to `str`.